workflows/files/ZWECONF.properties

```diff
@@ -625,3 +698,109 @@ components_explorer_mvs_enabled=true
 # Description:
 #  Check this option to enable USS explorer
 components_explorer_uss_enabled=true
+
+# useconfig_manager_enabled
+# Label: Enable Zowe configuration manager
+# Abstract: Should Zowe configuration manager be enabled?
+# Category: configManager
+# Description:
+#  Should Zowe configuration manager be enabled?
+useconfig_manager_enabled=true
+
+# config_manager_validation
+# Label: Zowe configuration manager validation type
+# Abstract: STRICT or COMPONENT-COMPAT validation type
+# Category: configManager
+# Description:
+#  STRICT=quit on any error, COMPONENT-COMPAT=if component missing schema, skip it with warning instead of quit
+# Choices: COMPONENT-COMPAT,STRICT
+config_manager_validation=COMPONENT-COMPAT
+
+# zowe_rbacProfileId
+# Label: Zowe Profile Identifier
+# Abstract: ID you use to separate multiple Zowe installs
+# Category: zowe
+# Description:
+#  ID when determining resource names used in RBAC authorization checks such as dataservices with RBAC expects this ID in SAF resources
+zowe_rbacProfileId=1
+
+# zowe_cookieId
+# Label: Zowe Cookie Identifier
+# Abstract: ID that can be used by servers that distinguish their cookies from unrelated Zowe installs
```

Comment:
ID that can be used by servers to distinguish their cookies from unrelated Zowe installs